### PR TITLE
Add fee estimate endpoint

### PR DESCRIPTION
**Pull request by sahith-narahari**
Closes: https://github.com/allinbits/demeris-backend/issues/239

### CODE REVIEW

File: api/tx/api.go [hunk 1]
```diff
@@ -177,3 +179,117 @@ func GetTicket(c *gin.Context) {
...
+
+		return
+	}
+
+	client, err := sdkservice.Client(chain.MajorSDKVersion())
+	if err != nil {
+		e := deps.NewError(
+			"chains",
+			fmt.Errorf("cannot retrieve sdk-service for version %s with chain name %v", chain.CosmosSDKVersion, chain.ChainName),
+			http.StatusBadRequest,
```
Comment (sgerogia) [inline api/tx/api.go]: Is this a 400 or a 500?

File: api/tx/api.go [hunk 2]
```diff
@@ -177,3 +179,117 @@ func GetTicket(c *gin.Context) {
...
+
+// GetTxFeeEstimate returns the estimated gas and fee price for specified chain.
+// @Summary estimates the gas and fees fot transaction.
+// @Tags Tx
+// @ID tx
+// @Description estimate transaction fees for the relevant chain.
+// @Param chainName path string true "chain name"
+// @Produce json
+// @Success 200 {object} TxFeeEstimateRes
+// @Failure 500,403 {object} deps.Error
```
Comment (sgerogia) [inline api/tx/api.go]: Error codes should reflect behaviour?

**Review by sgerogia — DISMISSED** [review]
Tests?

File: api/tx/api.go (hunk 2 above)
Comment (sahith-narahari) [reply api/tx/api.go]: addressed

File: api/tx/api.go (hunk 1 above)
Comment (sahith-narahari) [reply api/tx/api.go]: addressed

**Review by sgerogia — APPROVED** [review]
Couple of linting issues, otherwise feel free to merge

**Pull request merged**